docs/development.md

```diff
@@ -44,21 +44,30 @@ mv kustomize /usr/local/bin/
 ```
 ### Set Env variables
 ```shell
-export version = latest 
-export SIDECAR_IMG = quay.io/aneeshkp/cloud-event-proxy
-export  PRODUCER_IMG = quay.io/aneeshkp/cloud-native-event-producer
-export  CONSUMER_IMG = quay.io/aneeshkp/cloud-native-event-consumer
+export version=latest 
+export SIDECAR_IMG=quay.io/aneeshkp/cloud-event-proxy
+export  PRODUCER_IMG=quay.io/aneeshkp/cloud-native-event-producer
+export  CONSUMER_IMG=quay.io/aneeshkp/cloud-native-event-consumer
 ```
 
+### Setup AMQ Interconnect
+
+Install the `Red Hat Integration - AMQ Interconnect` operator in the `cloud-native-events` namespace from the OpenShift Web Console.
```

Comment:
We should able to deploy amqp to its own namespace. and use the full-service name to access,
amqp-interconnect.NAMESPCE.svc.cluster.local
Need to test this process.

Reply:
But this is okay for example

Reply:
will test this